### ISSUE TITLE
Failed to clone with wale

### ISSUE THREAD

**Issue by kidynamit** [rehive/charts#5]
Steps to replicate:
1. copy backup files from one bucket to another

```gsutil -m cp -r gs://rehive-core-pgbackup-production//spilo/patroni-rehive-core-patroni gs://rehive-core-pgbackup-ha-staging//spilo/production-backup```

2. Update values file to reflect new cloneWaleScope
3. Perform a new patroni deployment. *Check that the persistent volume is deleted.*

Results after deployment: 

```
lzop: <stdin>: not a lzop file
wal_e.blobstore.gs.utils WARNING  MSG: retrying WAL file fetch from unexpected exception
        DETAIL: The exception type is <class 'google.api_core.exceptions.Forbidden'> and its value is 403 GET https://www.googleapis.com/download/s
torage/v1/b/rehive-core-pgbackup-ha-staging/o/%2Fspilo%2Fproduction-backup%2Fwal%2Fwal_005%2F00000015.history.lzo?alt=media: ('Request failed with
status code', 403, 'Expected one of', <HTTPStatus.OK: 200>, <HTTPStatus.PARTIAL_CONTENT: 206>) and its traceback is   File "/usr/local/lib/python3.
6/dist-packages/wal_e/retries.py", line 87, in shim
            return f(*args, **kwargs)
          File "/usr/local/lib/python3.6/dist-packages/wal_e/blobstore/gs/utils.py", line 99, in download
            raise exc
          File "/usr/local/lib/python3.6/dist-packages/wal_e/blobstore/gs/utils.py", line 130, in write_and_return_error
            blob.download_to_file(stream)
          File "/usr/local/lib/python3.6/dist-packages/google/cloud/storage/blob.py", line 562, in download_to_file
            _raise_from_invalid_response(exc)
          File "/usr/local/lib/python3.6/dist-packages/google/cloud/storage/blob.py", line 1956, in _raise_from_invalid_response
            raise exceptions.from_http_status(response.status_code, message, response=response)
          There have been 2 attempts to fetch wal file gs://rehive-core-pgbackup-ha-staging/spilo/production-backup/wal/wal_005/00000015.history.lz
o so far.
        STRUCTURED: time=2019-01-29T15:41:33.796339-00 pid=261
```

\kind bug
\tag patroni

**Comment by kidynamit** [comment]
Related to rehive/rehive-core/issues/808 and zalando/patroni/issues/717

**Comment by tesserarius** [comment]
Resolved with `patroni-0.8.2`
